client/src/pages/UserSettings/components/SettingsListItem.tsx

```diff
@@ -1,46 +1,325 @@
-import { Divider, IconButton, ListItem, ListItemText } from '@mui/material';
+import {
+  Alert,
+  Divider,
+  IconButton,
+  Snackbar,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
 import EditIcon from '@mui/icons-material/Edit';
+import ClearIcon from '@mui/icons-material/Clear';
 import React from 'react';
-import { SettingsList, SecondaryText } from './styles';
+import { SettingStack, SecondaryText } from './styles';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+import axios, { AxiosError } from 'axios';
+import { updatePassword, updateUsername } from '../../../api/UserAPI';
+import SaveAltIcon from '@mui/icons-material/SaveAlt';
+import { useUser } from '../../../context/UserContext';
+import { theme } from '../../../assets/styles';
+import {
+  UpdatePasswordVals,
+  UpdateUsernameVals,
+} from '../../../interfaces/interfaces';
```

Comment:
nitpick, but can we organize those imports so that we have react/mui import chunk separated from our internal imports chunk so its easier to distinguish between the two.